tools/clang/lib/SPIRV/DeclResultIdMapper.cpp

```diff
@@ -3407,6 +3425,21 @@ SpirvVariable *DeclResultIdMapper::getBaseInstanceVariable(
   return baseInstanceVar;
 }
 
+SpirvVariable *DeclResultIdMapper::getBaseVertexVariable(
+    SemanticInfo *semantic, const hlsl::SigPoint *sigPoint, QualType type) {
+  assert(type->isSpecificBuiltinType(BuiltinType::Kind::Int) ||
+         type->isSpecificBuiltinType(BuiltinType::Kind::UInt));
+  auto *baseVertexVar = spvBuilder.addStageBuiltinVar(
+      type, spv::StorageClass::Input, spv::BuiltIn::BaseVertex, false,
+      semantic->loc);
+  StageVar var(sigPoint, *semantic, nullptr, type,
```

Comment:
```suggestion
  StageVar var(sigPoint, /* semaInfo= */, nullptr, type,
```

I wouldn't put the wrong semantic just to fill this.
Same for the location above, I'd probably use `{}` since this doesn't comes directly from a line of code.